mne/report.py

```diff
@@ -941,6 +942,21 @@ class Report(object):
         .. versionadded:: 0.21
     %(verbose)s
 
+    Attributes
+    ----------
+    initial_id : int
```

Comment:
think we can make this private safely. I doubt anyone ever uses it. It's for setting the "div id"

Reply:
Do private attributes require documentation?

Reply:
Private attributes do not need to be documented, but can be if there is something tricky about them that wouldn't be obvious from the code or the comments in the code